style.css

```diff
@@ -0,0 +1,69 @@
+    header {
+        position: fixed;
+        background-color: #ffffff;
+        width: 100%;
+        height: 70px;
+    }
+    
+    body {
+        background-color: black;
```

Comment:
Why it is black? I don't see that rule in design

Reply:
That's fixed!